openpype/hosts/maya/plugins/publish/validate_rig_out_set_node_ids.py

```diff
@@ -85,3 +85,50 @@ def repair(cls, instance):
                 continue
 
             lib.set_id(node, sibling_id, overwrite=True)
+
+    @classmethod
+    def get_node(cls, instance):
+        """Get target object nodes from out_SET
+
+        Args:
+            instance (str): instance
+
+        Returns:
+            list: list of object nodes from out_SET
+        """
+        return instance.data["rig_sets"].get("out_SET")
+
+
+class ValidateSkeletonRigOutSetNodeIds(ValidateRigOutSetNodeIds):
+    """Validate if deformed shapes have related IDs to the original shapes
+    from skeleton set.
+
+    When a deformer is applied in the scene on a referenced mesh that already
+    had deformers then Maya will create a new shape node for the mesh that
+    does not have the original id. This validator checks whether the ids are
+    valid on all the shape nodes in the instance.
+
+    """
+
+    order = ValidateContentsOrder
+    families = ["rig.fbx"]
+    hosts = ['maya']
+    label = 'Skeleton Rig Out Set Node Ids'
+    actions = [
+        openpype.hosts.maya.api.action.SelectInvalidAction,
+        RepairAction
+    ]
+    allow_history_only = False
```

Comment:
You should only redefine what you're actually changing. If the docstring should be the same, then remove that too. :) This goes for all the other plugins too that inherit like this.
```suggestion
    families = ["rig.fbx"]
    label = 'Skeleton Rig Out Set Node Ids'
```